data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,103 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition Folklore: Kimek"
+	minor
+	invisible
+	to offer
+		random < 30
+	source
+		attributes "kimek" "folklore"
+	on offer
+		conversation
+			`A Kimek flags you down when you enter the spaceport. "Busy are you, Captain?" He asks. "Taking place today, a cultural festival is, and participate in it, anyone can. A reenactment of the old 'Spring Siege' folk tale, it is. Play the part of one of the characters, anyone can."`
+			choice
+				`	"That sounds like fun, where is it being held?"`
+				`	"Sorry, I'm too busy right now for that."`
+					decline
+			`	He asks you to follow him, and you two board a train leading to another section of the city. You get off at the edge of a large park. There, you see that many sets of camps dot the savanna plains, and an equal number of wooden forts stare down from the top of a long hill. Each "set" of fort and camp is separated from the others by many patches of vegetation that go up the hill.`
+			`	You follow the Kimek for a while longer, until he finds you a pairing of camp and fort that have a vacant spot in either side. "Well Captain, choose a side to play as, you must. The hang of it, you will get, once started, the event has, but as a quick summary, take the 'crops' inside the fort before time runs out, the attacking team must. Prevent them from doing that, the defenders must. At the end, rewarded, the victors are."`
+			choice
+				`	"I'll join the attackers."`
+					goto att
+				`	"I'll join the defenders."`
+					goto def
+				`	"Hang on, if this is a folk tale, how come you're reenacting it as if any side could win?"`
+			`	He chitters, as if expecting the question. "Witnessed by a minstrel, the battle was. Travelling on fields like these, he was, before the fort, and the raider camp, he spotted in the distance. Hid himself in foliage close to the scene, he had, and writing and painting it as it happened, he was, but when reached its climax, the battle had, swept up by the wind, many flowers were, and block his vision they did. Once settled down the wind had, over, the battle was. Spring it was, so 'Spring Siege' became the story's title."`
+			`	Seeing you have more questions now, he continues. "Silly and nonsensical at times, the story is. Know that, we do, and that simply looking to entertain a crowd, the minstrel was, but with all forms of classic tales, is that not the case?" He chitters some more, and looks at the group up on the hill, and the other in the camp, before continuing. "Foreign as it might seem, participate in it, will you not? Perhaps like it, you might."`
```

Comment:
```suggestion
			`	Seeing you have more questions now, he continues. "Silly and nonsensical at times, the story is. Know that, we do, and that simply looking to entertain a crowd, the minstrel was, but with all forms of classic tales, is that not the case?" He chitters some more, and looks at the group up on the hill and the other in the camp, before continuing. "Foreign as it might seem, participate in it, will you not? Perhaps like it, you might."`
```